doc/doxygen-root/README.md

```diff
@@ -0,0 +1,60 @@
+# CBMC documentation
+
+This is the root of CBMC documentation.
+
+## Install doxygen
+
+Building this documentation requires `doxygen` and `graphviz`.
+
+* On MacOS,
+  we recommend installing with [brew](https://brew.sh/):
+  ```
+  brew install doxygen graphviz
```

Comment:
Does this need `pandoc` as well?

I tried without and got some errors?

Reply:
I've added pandoc to the instructions.  I omitted it because it is already installed on the GitHub ubuntu runner.  But good catch.